src/main/kotlin/net/irisshaders/lilybot/extensions/events/LogUploading.kt

```diff
@@ -17,14 +17,19 @@ import dev.kord.core.entity.Message
 import dev.kord.core.event.message.MessageCreateEvent
 import dev.kord.rest.builder.message.modify.actionRow
 import dev.kord.rest.builder.message.modify.embed
-import io.ktor.client.*
-import io.ktor.client.request.*
-import io.ktor.util.*
+import io.ktor.client.HttpClient
+import io.ktor.client.request.post
+import io.ktor.client.request.setBody
+import io.ktor.client.request.forms.FormDataContent
+import io.ktor.client.statement.readBytes
+import io.ktor.http.Parameters
 import kotlinx.datetime.Clock
+import kotlinx.serialization.decodeFromString
 import net.irisshaders.lilybot.utils.responseEmbedInChannel
 import java.io.ByteArrayInputStream
 import java.util.zip.GZIPInputStream
 import kotlin.time.ExperimentalTime
```

Comment:
We can remove this and the opt in at the top.

Reply:
Ignore this, it's fixed in the tags pr

Reply:
I don't understand what you mean by this, the import is necessary for line 189 and I didn't even add the opt in